### PR TITLE
Updating to work with INQ calculator

### PR DESCRIPTION
**Pull request by nkeilbart**
Had to change some minor parts in the calculator to allow the import of an Ase calculator included with the INQ code. Have a couple of example scripts as well. Updating some other formatting issues.

### CODE REVIEW

File: src/aiida_ase/calculations/ase.py [hunk 1]
```diff
@@ -239,7 +239,7 @@ def return_a_function(v):
         input_txt += f"atoms = ase.io.read('{self._input_aseatoms}')\n"
         input_txt += '\n'
         input_txt += f'calculator = custom_calculator({calc_argsstr})\n'
-        input_txt += 'atoms.set_calculator(calculator)\n'
+        input_txt += 'atoms.calc = calculator\n'
```
Comment (sphuber) [inline src/aiida_ase/calculations/ase.py:242]: Why is this change necessary?

File: examples/calculations/example_inq_relax.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+"""
+Example script of how to perform a SCF calculation with the python INQ
```
Comment (sphuber) [inline examples/calculations/example_inq_relax.py]:
```suggestion
Example script of how to perform a geometry optimization with the python INQ 
```

File: examples/calculations/example_inq_relax.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            'name': 'BFGS',
+            'run_args': {
+                'fmax': 0.02,
+                'steps': 50
+            }
+        }
+    }
+
+    # Running the calculation
+    settings = {}
```
Comment (sphuber) [inline examples/calculations/example_inq_relax.py]:
```suggestion
```

File: examples/calculations/example_inq_relax.py [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+    builder = AseCalculation.get_builder()
+    builder.code = load_code(CODE_NAME)
+    builder.structure = structure
+    builder.kpoints = kpoints
+    builder.parameters = orm.Dict(parameters)
+    builder.settings = orm.Dict(settings)
```
Comment (sphuber) [inline examples/calculations/example_inq_relax.py]:
```suggestion
```

File: examples/calculations/example_inq.py [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+                    'name': 'fermi-dirac',
+                    'width': 0.05
+                }
+            }
+        }
+    }
+
+    # Running the calculation using GPAW Python
+    settings = {}
+
```
Comment (sphuber) [inline examples/calculations/example_inq.py]:
```suggestion
```

File: examples/calculations/example_inq.py [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+    builder = AseCalculation.get_builder()
+    builder.code = load_code(CODE_NAME)
+    builder.structure = structure
+    builder.kpoints = kpoints
+    builder.parameters = orm.Dict(parameters)
+    builder.settings = orm.Dict(settings)
```
Comment (sphuber) [inline examples/calculations/example_inq.py]:
```suggestion
```

**Review by sphuber — CHANGES_REQUESTED** [review]
Thanks @nkeilbart

File: src/aiida_ase/calculations/ase.py (hunk 1 above)
Comment (nkeilbart) [reply src/aiida_ase/calculations/ase.py:242]: I was getting a warning that was being pulled into the results saying that it should be set like this now. I think it was still working but producing the warning.

**Review by nkeilbart — COMMENTED** [review]
All the changes look good.

**Comment by sphuber** [comment]
> All the changes look good.

Thanks @nkeilbart but the pre-commit and tests were failing. For future reference, it is recommended you install the pre-commit hook. You can do this by running
```
pip install -e .[pre-commit]
pre-commit install
```
inside your repo. This installs the dependencies and then installs the hook. The pre-commit hooks are then run automatically when you commit.

The tests were also failing because the reference files are out of date. This is easily fixed by running `pytest --force-regen`. This will update the reference files and you can then commit the changes.

**Review by sphuber — APPROVED** [review]
